math-doc/math/scribblings/math-special-functions.scrbl

```diff
@@ -56,11 +56,11 @@ The most general type @racket[Real -> (U Zero Flonum)] is used to generate
 @racket[lambert]'s contract when it is used in untyped code. Except for this discussion,
 this the only type documented for @racket[lambert].
 
-@section[#:tag "real-functions"]{Real Functions}
+@section[#:tag "complex-real-functions"]{Complex and real Functions}
 
-@defproc[(gamma [x Real]) (U Positive-Integer Flonum)]{
+@defproc[(gamma [x Number]) Number]{
```

Comment:
Could the result type be more specific? For instance, something including Float-Complex but not all complex numbers?

Reply:
Is this a question about the documentation, or making the type of my function more specific?